### PR TITLE
Remove rc<service> controls of systemd services (jsc#SLE-11624)

### PR DESCRIPTION
**Pull request by tbazant**
### Description
This update removes remains of rc<service> scripts that SUSE products used to control systemd services

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/apparmor_support.xml [hunk 1]
```diff
@@ -592,17 +592,18 @@ sock_type="raw" pid=30251 comm="ping"
   </sect2>
 
   <sect2 xml:id="sec-apparmor-support-trouble-syntax">
-   <title>How to Spot and fix &aa; Syntax Errors?</title>
+   <title>How to Spot and Fix &aa; Syntax Errors?</title>
```
Comment (lproven) [inline xml/apparmor_support.xml:595]: Does this _need_ the question mark?

If not, I suggest removal.

Comment (tbazant) [reply xml/apparmor_support.xml:595]: no, i'll remove it

Comment (tbazant) [inline xml/apparmor_support.xml:595]:
```suggestion
   <title>How to Spot and Fix &aa; Syntax Errors</title>
```

Comment (tbazant) [reply xml/apparmor_support.xml]: done

**Pull request merged**